xfields/ibs/_equilibrium.py

```diff
@@ -253,59 +253,78 @@ def compute_emittance_evolution(
+            - sr_ibs_eq_gemitt_x: final horizontal equilibrium geometric emittance converged to, in [m].
+            - sr_ibs_eq_gemitt_y: final vertical equilibrium geometric emittance converged to, in [m].
+            - sr_ibs_eq_gemitt_zeta: final longitudinal equilibrium geometric emittance converged to, in [m].
     """
-    # Returns
-    # -------
-    # xtrack.TwissTable
-    #     The convergence calculations results. The table contains the following
-    #     columns, as time-step by time-step quantities:
-    #         - time: time values at which quantities are computed, in [s]
-    #         - gemitt_x: horizontal geometric emittance values, in [m]
-    #         - gemitt_y: vertical geometric emittance values, in [m]
-    #         - gemitt_zeta: longitudinal geometric emittance values, in [m]
-    #         - Tx: horizontal IBS growth rate, in [s^-1]
-    #         - Ty: vertical IBS growth rate, in [s^-1]
-    #         - Tz: longitudinal IBS growth rate, in [s^-1]
-    #     The table also contains the following global quantities:
-    #         - damping_constants_s: radiation damping constants per second used for the calculations
-    #         - partition_numbers: damping partition numbers used for the calculations
-    #         - eq_gemitt_x: final horizontal equilibrium geometric emittance converged to, in [m]
-    #         - eq_gemitt_y: final vertical equilibrium geometric emittance converged to, in [m]
-    #         - eq_gemitt_zeta: final longitudinal equilibrium geometric emittance converged to, in [m]
-    #         - sr_eq_gemitt_x: horizontal equilibrium geometric emittance from synchrotron radiation used, in [m]
-    #         - sr_eq_gemitt_y: vertical equilibrium geometric emittance from synchrotron radiation used, in [m]
-    #         - sr_eq_nemitt_zeta: longitudinal equilibrium normalized emittance from synchrotron radiation used, in [m]
-    # ----------------------------------------------------------------------------------------------
-    # TODO: Perform check for valid value of emittance_constraint but no value of emittance coupling factor
-    # ----------------------------------------------------------------------------------------------
+    # fmt: off
+    # ---------------------------------------------------------------------------------------------
+    # Check for SR equilibrium emittances, damping constants and partition numbers in the TwissTable
+    _required_attrs = ["damping_constants_s", "partition_numbers", "eq_gemitt_x", "eq_gemitt_y", "eq_gemitt_zeta"]
+    if any(getattr(twiss, attr, None) is None for attr in _required_attrs):
+        LOGGER.error("Invalid TwissTable, does not have SR equilibrium properties. Did you configure radiation?")
+        raise AttributeError(
+            "The TwissTable must contain SR equilibrium emittances and damping constants. "
+            "Did you activate radiation and twiss with `eneloss_and_damping=True?`"
+        )
```

Comment:
Check that all these are here (we use each one of them at some point). Error message should be clear enough, let me know.